apps/andi/priv/repo/migrations/20200810140819_add_harvested_datasets.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Andi.Repo.Migrations.AddHarvestedDatasets do
+  use Ecto.Migration
+
+  def change do
+    alter table(:datasets) do
```

Comment:
I thought we weren't manipulating this table and just making a new one instead?

Reply:
my B. forgot to push the deleted migration